stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/MessageListView.kt

```diff
@@ -59,6 +58,11 @@ import kotlin.math.min
  * - The list_item_message template to use (perhaps, multiple ones...?)
  */
 public class MessageListView : ConstraintLayout, IMessageListView {
+
+    private companion object {
+        const val LOAD_MORE_TRESHOLD = 10
```

Comment:
Moved this into a code constant from resources, seems like we don't gain much by having it as a resource